shared/reducers/tracker.js

```diff
@@ -125,6 +129,11 @@ function updateUserState (state: TrackerState, action: Action): TrackerState {
     case Constants.onFollow:
       return {
         ...state,
+        lastTrack: {
```

Comment:
just making a fake lastTrack. maybe we should really do an identify or something but this is a short term fix